mne/viz/topomap.py

```diff
@@ -3015,3 +3015,77 @@ def _set_adjacency(adjacency, both_nodes, value):
 
     with warnings.catch_warnings(record=True):
         adjacency[both_nodes, both_nodes[::-1]] = value
+
+
+@fill_doc
+def plot_regression_weights(model, ch_type=None, vmin=None, vmax=None,
+                            cmap=None, sensors=True, colorbar=True, res=64,
+                            size=1, cbar_fmt='%.2E', show=True,
+                            show_names=False, title=None, mask=None,
+                            mask_params=None, outlines='head', axes=None,
+                            sphere=None, border=_BORDER_DEFAULT):
+    """Plot the regression weights of a fitted EOGRegression model.
+
+    Parameters
+    ----------
+    model : EOGRegression
```

Comment:
Needs to be put in `doc/conf.py` as a proper alias to `mne.preprocessing.EOGRegression`, it's why CircleCI complains